02Arrays/Array/Tester.swift

```diff
@@ -82,6 +151,18 @@ class Tester {
         [1, 4, 8, 4] -> [1, 2, 3, 2]
      */
```

Comment:
did this pass the test?

this will always return [1,2,3,4] of any n length array (if input array length is 4)